runtime/compiler/z/codegen/J9BCDTreeEvaluator.cpp

```diff
@@ -267,7 +267,7 @@ J9::Z::TreeEvaluator::udsl2pdEvaluator(TR::Node *node, TR::CodeGenerator *cg)
 
    if (!node->getOpCode().isSetSign())
       {
-      TR::LabelSymbol * startLabel = TR::LabelSymbol::create(cg->trHeapMemory(),cg);
+      TR::LabelSymbol * cFlowRegionStart = TR::LabelSymbol::create(cg->trHeapMemory(),cg);
       TR::LabelSymbol * doneLabel = TR::LabelSymbol::create(cg->trHeapMemory(),cg);
```

Comment:
According to https://github.com/eclipse/omr/issues/2833#issuecomment-432058904 this one should have been `cFlowRegionEnd` as it doesn't have an otherwise meaningful name. There are several occurrences of this. Do we need to be more consistent?

Also while we are in the area of modifying these names could you also change all the `TR::LabelSymbol::create` calls to `generateLabelSymbol` calls which doesn't need memory passed?

Reply:
I've tried to replace all the names of labels that have end ICF flags with `cFlowRegionEnd` where applicable in b34fad6, hopefully I haven't missed any. Regarding your second, I've replaced all calls to `TR::LabelSymbol::create` with `generateLabelSymbol` in 0a8b4ca.